edgelet/iotedge/src/init/import/mod.rs

```diff
@@ -20,6 +20,7 @@ use std::path::Path;
 
 use config::Config;
 
+use edgelet_core::{AZIOT_EDGED_CA_ALIAS, TRUST_BUNDLE_ALIAS};
```

Comment:
To be consistent, `TRUST_BUNDLE_ALIAS` should be `aziot-edged-trust-bundle` instead of `iotedge-trust-bundle` ?

Also, should `AZIOT_EDGED_CA_ALIAS` be `aziot-edged-ca` instead of `aziot-edged-device-ca` ?

Reply:
Yes - thanks for catching that inconsistency